src/engine/controls/bpmcontrol.cpp

```diff
@@ -733,18 +743,16 @@ double BpmControl::getBeatMatchPosition(
     if (!m_pBeats) {
         return dThisPosition;
     }
-    // Master buffer is always in sync!
-    if (getSyncMode() == SYNC_MASTER) {
-        return dThisPosition;
-    }
```

Comment:
```    
    if (getSyncMode() == SYNC_MASTER_EXPLICIT) {
        return dThisPosition;
    }
```
still not fails the test 

SYNC_MASTER_SOFT
fails EngineSyncTest.QuantizeImpliesSyncPhase Test

Reply:
ah ok, yes explicit master is always in sync yes